src/app/WriteHandler.cpp

```diff
@@ -634,6 +645,52 @@ Status WriteHandler::ProcessWriteRequest(System::PacketBufferHandle && aPayload,
     return status;
 }
 
+CHIP_ERROR WriteHandler::VerifyAttributeDataIBs(const TLV::TLVReader & aAttributeDataIBsReader, bool aIsGroupAction)
```

Comment:
maybe put this function in messageDef/AttributeDataIBs?

Reply:
please return status code directly?

Reply:
While checking AttributeDataIBs, I think this logic should live in the schema check functions.

However, it is enabled only with build flags (minor issue), and it will be coupled with the session (major issue, we allow missing endpoint id for group sessions, but not allow (for now) for unicasts)

Also, AttributePathIBs should be checked in different way for different transctions.

Updated the signature to return a IM Status instead of a CHIP_ERROR.